projects/packages/scheduled-updates/changelog/fix-scheduled-updates-cleanup

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: changed
+Comment: No functional changes
+
+
```

Comment:
This should be:
```
Significance: patch
Type: changed

Comment: No functional changes

```

Reply:
That's what `jetpack changelogger add` created 🤷

Reply:
Ok, not a problem. This changelog will be stripped away anyway. 🙂